students/rhamersky/Lesson_3/list_lab.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+# Description: This program will teach the user about Python list.
+# Developer: Ryan Hamersky
+# Date: 05/08/2018
+# Rev: A - 05/15/2018 add comments to the code.
+
+# -----Data Section-----
+l = ["Apples", "Pears","Oranges", "Peaches"]
+
+# -----Process Section-----
+# Series 1 #
+
+print()  # --> Formatting
+print("Series 1")
+print()  # --> Formatting
+
+print(l)  # --> Displays fruit list 'l'
+
+l.append(input("Please, add a fruit to the end of the list. ").title())  # --> Ask user for new fruit
+
+print(l)  # --> Displays new list to user.
+
+index = int(input("Please, enter a number to display the fruit. "))  # --> Receive number input from user.
+
+fruit = l[index-1]  # --> Finds fruit from the number the user inputted.
+
+print(fruit)  # --> Prints the fruit to the user.
+
+new_fruit = [input("Please, add another fruit to the beginning of the list. ").title()]   # --> Ask the user for a fruit.
+
+l = new_fruit+l  # --> Adds a new fruit to the beginning of the list.
+
+print(l)  # --> Prints new list.
+
+# Asks user for a fruit and inserts the fruit at the beginning of the list.
+l.insert(0,input("Please, add another fruit to the beginning of the list. ").title())
+
+print(l)  # --> Prints new list.
+
+# Prints all the fruits that begin with the letter 'P'.
+for fruits in l:
+    if "P" is fruits[0] or "p" is fruits[0]:
```

Comment:
you could do `if fruits[0].lower() == "p"` or `if fruits.lower().startswith('p')`

again, should be using `==` operator instead of `is`

is checks that 2 arguments refer to the same object, == checks that 2 arguments have the same value.  see:
```
>>> 1000 is 10**3
False
>>> 1000 == 10**3
True
```

you would use `is` when checking for None, True or False -> `var is None`, `var is True`.. and even then it is more pythonic to check for "truthiness" or "falsiness" -> `if var` or `if not var` unless there is specific distinction for a var to be boolean type.

Reply:
Thanks for this. I never really knew between the 'is' and '=='.  

Ah thanks for the tip with the method startswith.